android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/MainActivity.java

```diff
@@ -259,43 +254,45 @@ protected void onResume() {
     KMKeyboardDownloaderActivity.addKeyboardDownloadEventListener(this);
     PackageActivity.addKeyboardDownloadEventListener(this);
 
+    CheckInstallReferrer.checkGooglePlayInstallReferrer(this, context);
```

Comment:
This is probably not required but it doesn't hurt; the check on start is most likely sufficient.